__tests__/app.test.js

```diff
@@ -212,10 +213,147 @@ describe('PATCH requests testing', () => {
                 .expect(400)
                 .then(({ body: { msg } }) => {
 
-                    console.log(msg, "MSSSSG");
                     expect(msg).toEqual("Bad request");
                 })
 
         })
     });
 });
+
+// ------ POST - request testing------
+
+describe('POST request testing', () => {
+    describe('POST - /api/articles/:article_id/comments', () => {
+        // object to send as body in tests for this endpoint
+        const requestConntentToPost =
+        {
+            username: 'lurker',
+            body: 'Just looking'
+        }
+        test('Request body accepts: an object with the following properties: username, body. Responds with: the posted comment', () => {
+
+
+            return request(app)
+                .post("/api/articles/1/comments")
+                .send(requestConntentToPost)
+                .expect(200)
+                .then(({ body: { comment } }) => {
+                    expect(comment).toEqual(expect.objectContaining({
+                        comment_id: expect.any(Number),
+                        votes: 0,
+                        author: 'lurker',
+                        body: 'Just looking',
+                        created_at: expect.any(String),
+                        article_id: 1
+                    }))
+                })
+        });
+
+        test('Error 404 when wrong path been passed ', () => {
+            return request(app).post('/api/articles/3/not-valid-path')
+                .send(requestConntentToPost)
+                .expect(404)
+                .then((({ body: { msg } }) => {
+
+                    expect(msg).toEqual("Path not found");
+                }))
+
+        })
```

Comment:
We only need one "global" test for an invalid path, rather than one for each endpoint